<launch>
  <!-- Run the node which converts Laser info to PointCloud -->
  <node pkg="laser_filters" type="scan_to_cloud_filter_chain"
      name="laser_to_cloud">
    <rosparam command="load" file="$(find tw04_pcl)/laser_config.yaml" />
    <param name="high_fidelity" value="true" />
    <param name="target_frame" type="string" value="robot_0/odom" />
    <remap from="scan" to="robot_0/base_scan" />
    <remap from="/cloud_filtered" to="robot_0/cloud_filtered"/>
  </node>

  <!-- Run our application -->
<!--  <node pkg="tw04_pcl" type="tw04_pcl" name="tw04_pcl" required="true" cwd="node"/> -->
</launch>
